### PR TITLE
Annotate answer choices by order displayed, even with randomization

### LINKED ISSUE #172: Verify that the associated question number is correct
Might always be recording the number from the array, not the number displayed.

### PR DESCRIPTION
**Pull request by adrifelt**
All of the answer choices should be annotated based on what order they were displayed in. For example: "02-yes" for "yes" as the third answer choice. This should be true even if the answers are randomized.

Previously: the number was wrong if randomization
Now: number is correct even with randomization

Fixes #172

### CODE REVIEW

**Comment by adrifelt** [comment]
meacer ptal

File: extension/surveys/survey-generator.js [hunk 1]
```diff
@@ -276,13 +268,16 @@ ScaleQuestion.prototype.setAttributes = function(attributes) {
 ScaleQuestion.prototype.makeSingleRow =
     function(horizontal, questionName, reverse, showLabels) {
   var container = document.createElement('div');
-  var scaleElements = [];
-  for (var i = 0; i < this.scale.length; i++) {
+  var reversedScale = reverse ?
+      flipArray(
```
Comment (meacer) [inline extension/surveys/survey-generator.js]: I was looking at |flipArray|. It seems to duplicate the last item first by reversing the original array and then adding the last item again when |anchorLast| is true. Is that what you originally intended?

File: extension/background.js [hunk 2]
```diff
@@ -477,6 +477,7 @@ chrome.experienceSamplingPrivate.onDecision.addListener(recordEvent);
 function handleCompletedSurvey(message) {
   if (message[constants.MSG_TYPE] !== constants.MSG_SURVEY)
     return;
+  console.log(JSON.stringify(message));
```
Comment (meacer) [inline extension/background.js]: Did you mean to keep this log?

File: extension/surveys/survey-generator.js (hunk 1 above)
Comment (meacer) [inline extension/surveys/survey-generator.js]: Nope, I was reading it wrong. It works fine.

**Comment by meacer** [comment]
Lgtm

File: extension/background.js (hunk 2 above)
Comment (adrifelt) [inline extension/background.js]: Nope :P

**Pull request merged**